### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by JcMinarro**
Remove dead code

### CODE REVIEW

**Review by adasiewiczr — APPROVED** [review]
LGTM

File: library/src/main/java/com/getstream/sdk/chat/enums/QueryOptions.java [hunk 1]
```diff
@@ -1,4 +0,0 @@
-package com.getstream.sdk.chat.enums;
-
-public class QueryOptions {
-}
```
Comment (zsmb13) [inline library/src/main/java/com/getstream/sdk/chat/enums/QueryOptions.java:4]: 🤔

**Review by zsmb13 — APPROVED** [review]
The most satisfying kind of PR 😄  👍

File: library/src/main/java/com/getstream/sdk/chat/enums/QueryOptions.java (hunk 1 above)
Comment (JcMinarro) [reply library/src/main/java/com/getstream/sdk/chat/enums/QueryOptions.java:4]: :rofl: I had the same thought on it

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**